config/example-hybrid-corexy.cfg

```diff
@@ -0,0 +1,79 @@
+# This file is an example config file for hybrid corexy style printers also
```

Comment:
When adding an "example-" config file, it's a good idea to also add it to test/klippy/printers.test regression test tool.  See https://www.klipper3d.org/Example_Configs.html